notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
+
+Below the fold we should introduce concepts one at a time, each with visual metaphor and take-aways:
+
+1) GraphQL describes what a client needs in terms of how client developers think about data.
+
+* If you're familiar with JSON, GraphQL is easy to learn and understand.
+* It's easy to anticipate the shape of the result for any query.
+* GraphQL only sends what you ask for, nothing more or less.
+
+2) GraphQL queries can access many "resources" in a single network request.
+
+* A query can access properties of not just one object, but of many related objects.
+* Compared to REST, GraphQL requires much less network activity.
+
+3) GraphQL services provide a type system.
+
+* Every `{ }` corresponds to an object of a particular type, and every type describes the fields available.
+* GraphQL only runs queries that make sense.
+* Tools and IDEs can make editing queries easy via type-aheads.
+  * GraphiQL is a free tool that you can use.
+* Type system defines descriptions, making it easy to keep documentation up to date.
+
+4) GraphQL is composable via fragments.
+
+* Fragments describe a portion of some Type to be queried.
+* Fragments are often used next to View code where data is used.
+* Fragments are composed together to create full queries.
+
+5) GraphQL makes backwards-compatible APIs easy.
+
+* Server does not need to worry about concerns of any particular client, only the complete set of capabilities. Clients are responsible for the data they receive.
+* Because GraphQL only sends what you ask for, new capabilities can be introduced via new fields on types with no impact on existing queries.
+* Old capabilities can be marked "deprecated" with no impact on existing queries.
+* Far fewer reasons for versioning your API when using GraphQL.
+
+6) GraphQL queries are answered by simple functions on your server.
+
+* GraphQL is not backed by any database technology, just like REST.
+* Every field on each type is represented by a function for retrieving that data.
+* A GraphQL executor will call your functions and use as much concurrency as possible.
+* Possibilities are endless for connecting to your server's existing data model.
```

Comment:
Perhaps "it's easy to write a GraphQL API that connects to your existing data model" rather than "there is an endless number of possible implementations you will have to decide between"